cats/src/io/github/iltotore/iron/cats.scala

```diff
@@ -93,3 +91,6 @@ private object IronCatsInstances:
   inline given [A, B](using inline ev: Semilattice[A]): Semilattice[A :| B] = ev.asInstanceOf[Semilattice[A :| B]]
   inline given [A, B](using inline ev: Show[A]): Show[A :| B] = ev.asInstanceOf[Show[A :| B]]
   inline given [A, B](using inline ev: UpperBounded[A]): UpperBounded[A :| B] = ev.asInstanceOf[UpperBounded[A :| B]]
+
+private trait IronCatsLowPriority0:
```

Comment:
Why a `0` in the name? It does not seem necessary since we only have a single trait